NEWS.md

```diff
@@ -1,9 +1,21 @@
+# linelist 0.0.38.9000
+
+* `top_values()` no longer throws a spurious warning when the levels in the 
+  subset data are identical to the levels in the full data (#96)
+
+# linelist 0.0.37.9000
+
+* `top_values()` gains a new `subset` argument that allows the user to retain
```

Comment:
thanks for adding this (I keep forgetting bumping versions up); should this be a bump in minor rather than patch, as it is a new feature rather than a bug fix?

Reply:
I've been only bumping the patch in linelist since this is still the development phase (If we did it major, minor, patch, we would be at something like version 3.4.9). Once it's released to CRAN, we should bump to 0.1.0 and keep on with major, minor, patch versioning.

Reply:
understood, thanks